### PR TITLE
Configurable admin username and passsword for Openshift enterprise in Nuage.

### PR DESCRIPTION
**Pull request by abhat**
This change is to allow users to configure admin username and password for Nuage enterprise created corresponding to the openshift cluster.

### CODE REVIEW

**Comment by detiber** [comment]
@abutcher ptal

File: roles/nuage_master/vars/main.yaml [hunk 1]
```diff
@@ -15,3 +15,6 @@ nuage_ca_master_rest_server_key: "{{ nuage_mon_rest_server_crt_dir }}/nuageMonSe
 nuage_ca_master_rest_server_crt: "{{ nuage_mon_rest_server_crt_dir }}/nuageMonServer.crt" 
 
 nuage_master_crt_dir : /usr/share/nuage-openshift-monitor
+cspadminpasswd: ''
+adminsusername: 'admin'
+adminuserpasswd: 'admin'
```
Comment (abutcher) [inline roles/nuage_master/vars/main.yaml]: Could we add `nuage_master_` to the front of these variable names so we can quickly know which role makes use of them?

**Comment by abhat** [comment]
Updated the review with changes to address the comments from @abutcher

**Comment by detiber** [comment]
lgtm @brenton 

@abhat as an aside, would it be possible to get some type of documentation added to openshift-ansible or at least a link to the Nuage documentation? I've had a few people ask me about how the integration works and would like to be able to easily point future requests to the right info.

**Comment by abutcher** [comment]
aos-ci-test

**Comment by abutcher** [comment]
@brenton

**Pull request merged**